src/schema/geneInteraction.yaml

```diff
@@ -0,0 +1,467 @@
+      - mint
+      - pdbe
+      - rcsb pdb
+      - WB
+      - wwpdb
+    slot_usage:
+      id:
+        description: >-
+          The unique primary identifier for the interaction. This should be the source
+          (curation) database identifier, or if that is not available then the aggregation
+          database identifier (e.g. IMEx)
+      cross_references:
+        description: >-
+          Additional identifier(s) of the interaction annotation other than the primary id.
+      subject:
+        range: Gene                   # is this declaration needed since it is already made in the parent class?
+        values_from:
+          - alliance
+      predicate:
+        subproperty_of: interacts_with
+        symmetric: true
+      relation:
+        values_from:
+          - RO
+          - mi
+      object:
+        range: Gene                   # is this declaration needed since it is already made in the parent class?
+        values_from:
+          - alliance
+      data_provider:
+        range: uriorcurie
+        description: >-
+          The database that curated the interaction. e.g. BioGRID
+        values_from:
+          - mi
+#        exact_mappings: # need to determine how to specify that the values should be an ontology class or descendant
+#          - mi:0461
+        required: true
+        multivalued: false
+      association_publications:
+        description: >-
+          The reference from which the interaction was annotated.
+        multivalued: false
+    exact_mappings:
+      - biolink:pairwise_gene_to_gene_interaction
+
+  GeneMolecularInteraction:
+    description: >-
+      A physical molecular interaction between gene products (e.g. protein-protein
+      interactions or protein-RNA interactions) or between genes and DNA-binding
+      factors (e.g. protein-DNA interactions)
+    is_a: GeneInteraction
+    slots:
+      aggregation_database
+      detection_method
+    slot_usage:
+      aggregation_database:
+        values_from:
+          - mi
+#        exact_mappings: # need to determine how to specify that the values should be an ontology class or descendant
+#          - mi:0461
+        description: >-
+          The database that collected the interaction annotation from another resource and
+          aggregated it into a consolidated resource. e.g. IMEx
+      detection_method:
+        values_from:
+          - mi
+#        exact_mappings: # need to determine how to specify that the values should be an ontology class or descendant
+#          - mi:0001
+      predicate:
+        subproperty_of: molecularly_interacts_with
+        symmetric: true
+      relation:
+        values_from:
+          - RO
+          - mi
+    exact_mappings:
+      - biolink:pairwise_molecular_interaction
+
+  GeneGeneticInteraction:
+    description: >-
+      A genetic interaction between genes (e.g. phenotypic suppression)
+    is_a: GeneInteraction
+    slots:
+      interactor_A_genetic_perturbation
```

Comment:
need dashes here in your slots collection

Reply:
Thanks for catching that, fixed on my branch (will push soon).